source/chapters/library.rst

```diff
@@ -828,3 +828,80 @@ different location. Select :guilabel:`Use Default Library` to reset.
 
 .. seealso:: External libraries can be disabled under
              :menuselection:`Preferences --> Library`.
+
+Removing tracks from the library
+================================
+
+Removing tracks from the Mixxx library will **not** physically delete them from
+your drive. However it does delete extra metadata Mixxx might have (such as
+hotcues and the beatgrids), and removes links to playlists or crates.
```

Comment:
I am not sure if links to playlists are removed, if you hide a track.
At least we had a version where unhide restores the playlist and crate membership.